build/en/docs/index.njk

```diff
@@ -118,7 +128,12 @@ rootClass: docs
     <p>
       Click outside the image to exit background placement.
     </p>
-    <h2 id="Masking-cropping">Masking & cropping</h2>
+    <h2 id="Add-and-manage-page-layouts">Create &amp; manage page layouts</h2>
```

Comment:
We don't really support _creating_ page layouts yet. Planned for April.

Should we omit this for now?

Reply:
I've updated it to "Using page layouts", just to highlight they're there :)